client/layout/guided-tours/config-elements/step.js

```diff
@@ -321,7 +325,7 @@ export default class Step extends Component {
 
 		return (
 			<Card className={ classNames( ...classes ) } style={ style }>
-				{ children }
+				<ContentComponent translate={ translate } />
```

Comment:
I've never seen a render prop called in jsx form.
I had to do a double take looking for waht `ContentComponent` was -- but really it is the render function supplied as the 'children'.